mkosi/kmod.py

```diff
@@ -16,6 +16,13 @@
 from mkosi.util import chdir, parents_below
 
 
+def is_valid_modulesd(modulesd: Path) -> bool:
+    # Check whether a provided kernel modules directory is valid
+    return modulesd.is_dir() and (
+        (modulesd / "modules.dep").exists() or (modulesd / "modules.dep.bin").exists()
+    )
```

Comment:
Can you move this into initrd.py as well?

Reply:
Sure, done.